@here/harp-mapview/lib/geometry/overlayOnElevation.ts

```diff
@@ -6,8 +6,17 @@
 
 import { GeometryKind } from "@here/harp-datasource-protocol";
```

Comment:
I am a bit unhappy with the name "overlay" here. Does it really fit? Is this an actual overlay, or is the height of the label object just modified by the terrain? I would suggest to find a name that explains better what is being done here, otherwise we have to explain why this overlay is not the kind overlay the rest of the world is using, it just "overlays", which is terrible English.